android/clientlib-ktx/src/main/java/com/solana/mobilewalletadapter/clientlib/TransactionParams.kt

```diff
@@ -2,10 +2,30 @@ package com.solana.mobilewalletadapter.clientlib
 
 import com.solana.mobilewalletadapter.common.ProtocolContract
 
+@Deprecated(
+    "RpcCluster is deprecated as of MWA 2.0",
+    replaceWith = ReplaceWith("Use the Blockchain object for multi-chain support"),
+    DeprecationLevel.WARNING
+)
 sealed class RpcCluster(val name: String) {
     object MainnetBeta : RpcCluster(ProtocolContract.CLUSTER_MAINNET_BETA)
     object Testnet : RpcCluster(ProtocolContract.CLUSTER_TESTNET)
     object Devnet : RpcCluster(ProtocolContract.CLUSTER_DEVNET)
+    class Custom(name: String) : RpcCluster(name)
+}
+
+sealed class Blockchain(
+    val name: String,
+    val cluster: String
+) {
+    val fullName
+        get() = "$name:$cluster"
```

Comment:
note that cluster != chain id. we used `mainnet-beta` for the miannet cluster but we use `solana:mainnet` for the mainnet chain id.